ios-contact-manager/ios-contact-manager/Controller/ContactEditViewController.swift

```diff
@@ -7,28 +7,36 @@
 
 import UIKit
 
-final class ContactEditViewController: ContactCreateViewController {
-    override func configureViewComponents() {
-        self.ageTextField.keyboardType = .numberPad
-        self.phoneNumberTextField.keyboardType = .phonePad
-        
-        if let contact = contact {
-            super.contact = contact
-            self.nameTextField.text = contact.name
-            self.ageTextField.text = "\(contact.age)"
-            self.phoneNumberTextField.text = contact.phoneNumber
-        }
+final class ContactEditViewController: ContactFormViewController {
+    private let temporaryContainer: TemporaryContainer
+    
+    init?(coder: NSCoder, contact: Contact) {
+        self.temporaryContainer = TemporaryContainer(contact: contact)
+        super.init(coder: coder)
     }
     
+    required init?(coder: NSCoder) {
+        fatalError("Edit View Controller 생성 실패")
+    }
+    
+    override func configureViewComponents() {
+        guard let contact = self.temporaryContainer.contact else { return }
+        self.nameTextField.text = contact.name
+        self.ageTextField.text = "\(contact.age)"
+        self.phoneNumberTextField.text = contact.phoneNumber
+        super.configureViewComponents()
+    }
     override func touchSaveBarButton(_ sender: UIBarButtonItem) {
```

Comment:
```suggestion

    override func touchSaveBarButton(_ sender: UIBarButtonItem) {
```

=> 위의 메서드와 한칸씩 떨어져 있어야 보기 좋습니다 👍